tests/unit/test_create_notebook.py

```diff
@@ -54,20 +54,22 @@ def test_create_notebook_success(
     mock_resource_handler_instance = MagicMock()
     mock_resource_handler.return_value = mock_resource_handler_instance
 
-    # Mock away the check on existing notebooks
-    with patch("dss.create_notebook.does_notebook_exist", return_value=False):
-        # Mock wait_for_deployment_ready
-        with patch(
-            "dss.create_notebook.wait_for_deployment_ready"
-        ) as mock_wait_for_deployment_ready:
-            # Call the function to test
-            create_notebook(
-                name=notebook_name, image=notebook_image, lightkube_client=mock_client_instance
-            )
+    # Mock away the check on existing dss pvc
+    with patch("dss.create_notebook.does_dss_pvc_exist", return_value=True):
```

Comment:
Do we know what python version we want to require here?  If we say python>=3.10, we can [unnest these with blocks](https://stackoverflow.com/a/50958431)

Reply:
updated, given that we agreed in matrix on python>=3.10